students/11th/haejunkim/westagram/Post/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db   import models
+from User.models import User
+
+class Post(models.Model):
+    user      = models.ForeignKey(User, on_delete = models.CASCADE)
+    content    = models.TextField(max_length = 300)
+    image_url  = models.URLField()
+    created_at = models.DateTimeField(auto_now_add = True)
+
+    class Meta:
+        db_table = "post"
```

Comment:
일반적으로 테이블 이름은 복수형을 사용합니다.
```suggestion
        db_table = "posts"
```